packages/components/form/src/form.vue

```diff
@@ -138,7 +139,12 @@ const validateField: FormContext['validateField'] = async (
     const invalidFields = e as ValidateFieldsError
 
     if (props.scrollToError) {
-      scrollToField(Object.keys(invalidFields)[0])
+      // form-item may be dynamically rendered based on the judgment conditions, and the order in invalidFields is uncertain.
+      // Therefore, the first form field with an error is determined by directly looking for the rendered element.
+      if (formRef.value) {
+        const formItem = formRef.value!.querySelector('.is-error')
```

Comment:
There is another problem that bothers me. Currently, the location of the form item with an error is found through the `querySelector` api. If the user happens to manually add the corresponding class to other elements, it may cause an error in finding the element(Although the possibility is relatively small.).

I can't think of a more reasonable way to avoid this situation.